ivy_tests/test_ivy/test_frontends/test_jax/test_nn/test_non_linear_activations.py

```diff
@@ -699,7 +701,7 @@ def test_jax_swish(
 @handle_frontend_test(
     fn_tree="jax.nn.hard_swish",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
+        available_dtypes=helpers.get_dtypes("numeric"),
```

Comment:
This should be `float_and_complex`, not `numeric`. `hardswish` throws errors when passed integer values

Reply:
Done